blocklearning/lib/app/data/provider/storage_provider_keys.dart

```diff
@@ -0,0 +1,4 @@
+abstract class StorageProviderKeys {
```

Comment:
Why we use some smsDateKey? We won't use it!

Reply:
It's for exaple using providers and getx storage